src/components/TextInput/TextInput.js

```diff
@@ -206,6 +208,21 @@ class TextInput extends React.Component<Props, State> {
     });
   };
 
+  handleRNFocus = () => {
+    setTimeout(async () => {
+      this.multilineInputField._root.focus();
+      await this.setState({
```

Comment:
As I remember `setState()` doesn't return anything, so there is no need to write `await` before it